empathylink.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
+    settings = get_settings
+    @buddies = settings.hunting_buddies
+    @use_hodierna = DRC.get_settings.hashodierna
```

Comment:
This should reuse `settings`, not make another call to `get_setttings`

Reply:
Right.

Reply:
Changed.